Update Hypre git location
Patch by Simon Branford, University of Birmingham
--- config/BuildSystem/config/packages/hypre.py.orig	2021-01-04 09:51:25.200103000 +0000
+++ config/BuildSystem/config/packages/hypre.py	2021-01-04 09:52:01.539715000 +0000
@@ -5,7 +5,7 @@
   def __init__(self, framework):
     config.package.GNUPackage.__init__(self, framework)
     self.gitcommit = 'v2.15.1'
-    self.download  = ['git://https://github.com/LLNL/hypre','https://github.com/LLNL/hypre/archive/'+self.gitcommit+'.tar.gz']
+    self.download  = ['git://https://github.com/hypre-space/hypre','https://github.com/hypre-space/hypre/archive/'+self.gitcommit+'.tar.gz']
     self.functions = ['HYPRE_IJMatrixCreate']
     self.includes  = ['HYPRE.h']
     self.liblist   = [['libHYPRE.a']]
